javascript/packages/orchestrator/src/test-runner/index.ts

```diff
@@ -34,11 +35,11 @@ export async function run(
   provider: string,
   inCI = false,
   concurrency = 1,
-  silent = false,
+  logType: LogType = "table",
   runningNetworkSpecPath: string | undefined,
   dir: string | undefined,
 ) {
-  setSilent(silent);
+  logType && setLogType("silent");
```

Comment:
```suggestion
  logType && setLogType(logType);
```
I think we should set the `logType` arg and not `silent` here rigth?